system-int-test/src/cmr/system_int_test/system.clj

```diff
@@ -41,7 +41,7 @@
              ;; A map of the components (echo, elastic, db, and message queue) to whether they are
              ;; in-memory or external
              :component-type-map component-type-map}]
-    (transmit-config/system-with-connections sys [:echo-rest :search :access-control :urs])))
+    (transmit-config/system-with-connections sys [:echo-rest :search :access-control :urs :ingest])))
```

Comment:
Ah, @chris-durbin: I think this is why I was getting those weird inconsistencies with ingest ACLs in the sys-int tests (but not in the dev system). I think int tests previously worked because they hit the dev-system's HTTP endpoints ... maybe?